CONTRIBUTING.md

```diff
@@ -177,24 +177,43 @@ exposed.
 
 ## Pull Requests
 
-The master development branch is `master`.
-Branch names should be prefixed with the author, eg. `name/feature-x`.
+If you have write access to the ibc-rs repo, you can directly branch off of `master`.
+This makes it easier for project maintainers to directly make changes to your
+branch should the need arise.
 
-Pull requests are made against `master`
-and are squash merged into master.
+Branch names should be prefixed with the author's name followed by a short description
+of the feature, eg. `name/feature-x`.
 
-PRs must:
+Pull requests are made against `master` and are squash-merged into master.
 
-- make reference to an issue outlining the context.
-- update any relevant documentation and include tests.
+PRs must:
+- make reference to an issue outlining the context
+- update any relevant documentation and include tests
 - add a corresponding entry in the `.changelog` directory using `unclog`,
   see the section above for more details.
 
-Pull requests should aim to be small and self contained to facilitate quick
+Pull requests should aim to be small and self-contained to facilitate quick
 review and merging. Larger change sets should be broken up across multiple PRs.
 Commits should be concise but informative, and moderately clean. Commits will be squashed into a
 single commit for the PR with all the commit messages.
 
+In order to help facilitate the PR review process, tag *one* person as the
+reviewer of the PR. If you are unsure of who to tag, your point of contact on
+the ibc-rs team is always a natural candidate; they'll make sure that the PR gets
+reviewed by whomever is most appropriate to review it. It also helps to notify
+the person whom you tagged as reviewer through direct means, such as through
+Slack or Discord, as it is easy for GitHub notifications to get lost or buried.
+
+## Reviewing Pull Requests
+
+If you're tagged as the reviewer of a PR, you are responsible for seeing it
+through to completion. This includes shepherding the work, helping to ensure the
+PR doesn't get blocked and/or bogged down, and getting it merged as per the contribution guidelines.
```

Comment:
I think we can make this sentence stronger. WDYT?

> In case the PR author is away or unavailable, the responsibility of PR reviewer additionally includes fixing issues with the PR and taking any decisions needed to get it merged.